components/DetailPageSidebar/DetailPageSidebar.story.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { action, storiesOf } from '@storybook/react';
+import DetailPageSidebar from './DetailPageSidebar';
+import { Notification } from 'carbon-components-react';
+
+
+const detailPageSidebarProps = {
+  
+    artifact: {
+        name: "name",
```

Comment:
We use single-quotes for javascript, but this might be because I didn't set up linting properly on this repository - I'll take a look through it